codecov.yml

```diff
@@ -1,7 +1,16 @@
 coverage:
   status:
     project:
+      # no leniency on `main`
+      main:
+        branches:
+          - main
```

Comment:
maybe this should be the merge-queue branch instead? but they don't support regex here :/

Reply:
```suggestion
        branches:
          - main
          - gh-readonly-queue/main/*
```